src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/SuggestedAction.cs

```diff
@@ -117,7 +118,7 @@ private async Task InvokeAsync()
                 using var context = SourceProvider.UIThreadOperationExecutor.BeginExecute(
                     EditorFeaturesResources.Execute_Suggested_Action, CodeAction.Title, allowCancellation: true, showProgress: true);
                 using var scope = context.AddScope(allowCancellation: true, CodeAction.Message);
-                await this.InnerInvokeAsync(new UIThreadOperationContextProgressTracker(scope), context.UserCancellationToken).ConfigureAwait(false);
+                await this.InnerInvokeAsync(scope.GetCodeAnalysisProgress(), context.UserCancellationToken).ConfigureAwait(false);
```

Comment:
similar to before, this is how we 'bridge' from the actual progress UI the platform provides for us, to the simple `IProgress<CodeAnalysisProgress>` api that our apis will then use.  This keeps them unaware of how the end UI will actually appear and gives a simple API to use instead.